diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -7,21 +9,85 @@ use crate::serialize::{self, IsNull, Output, ToSql};
 use crate::sql_types::{Date, Time, Timestamp, TimestamptzSqlite};
 use crate::sqlite::Sqlite;
 
-const SQLITE_DATE_FORMAT: &str = "%F";
+const DATE_FORMAT: &str = "%F";
```

Comment:
Can we add a comment here that these formats need to be in sync with that one in the `time` support module (and vice versa)?

Reply:
Even better to make this comment doccomment and put a direct intradoc reference to the corresponding constant(?), then editors, like VS Code, will able to create a clickable link to it